src/components/Details/Details.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+import {
+  Detail, FlexDetails, TitleCountry, TextPop, LeftRight, TitleDetails, Text,
+} from './details.styled';
+
+const Details = () => {
+  const countries = useSelector((state) => state.reduceData);
+  const { country } = useParams();
+  const filterCountries = countries.filter((country) => country.continent === 'Europe');
+  const countryName = filterCountries.filter((item) => item.name === country);
+
+  return (
+    <div className="details">
+      <Header />
+      {countryName.map((country, index) => (
+        // eslint-disable-next-line react/no-array-index-key
```

Comment:
- [ ] Let's try and fix Linters issues instead of disabling them. Linters are great tools that analyze source code to flag programming errors, bugs, and stylistic errors.